changes/ee/feat-10409.en.md

```diff
@@ -1 +0,0 @@
-Add support for [Protocol Buffers](https://protobuf.dev/) schemas in Schema Registry.
```

Comment:
Is this "old"?  I think it's not included in the e5.0.3 alpha changelog, and was merged very recently to `master` only. :thinking:

Reply:
Indeed, I just ran the script and didn't check if it's valid.